src/test/java/org/littleshoot/proxy/ProxyWebUiTest.java

```diff
@@ -0,0 +1,96 @@
+package org.littleshoot.proxy;
+
+import static org.junit.Assert.*;
+import static org.hamcrest.Matchers.*;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.netty.handler.codec.http.DefaultFullHttpResponse;
+import io.netty.handler.codec.http.DefaultHttpResponse;
+import io.netty.handler.codec.http.HttpHeaders;
+import io.netty.handler.codec.http.HttpHeaders.Names;
+import io.netty.handler.codec.http.HttpHeaders.Values;
+import io.netty.handler.codec.http.HttpObject;
+import io.netty.handler.codec.http.HttpRequest;
+import io.netty.handler.codec.http.HttpResponse;
+import io.netty.handler.codec.http.HttpResponseStatus;
+import io.netty.handler.codec.http.HttpVersion;
+
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Test;
+import org.littleshoot.proxy.impl.DefaultHttpProxyServer;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+
+public class ProxyWebUiTest {
+
+    private static final String REDIRECT_PATH = "/";
+
+    private HttpFiltersAdapter answerRedirectFilter(HttpRequest originalRequest) {
+        return new HttpFiltersAdapter(originalRequest) {
+            @Override
+            public HttpResponse clientToProxyRequest(HttpObject httpObject) {
+                HttpResponse answer = new DefaultHttpResponse(HttpVersion.HTTP_1_1, HttpResponseStatus.FOUND);
```

Comment:
One thing to note here is that this method will return a new HTTP response when the client sends the HttpRequest, then return a new HttpResponse when the client sends the first HttpContent (if the request is a POST, for example). If the request is a GET, I believe Netty still sends an EmptyLastHttpContent through the pipeline after the HttpRequest, so this filter method would still return a duplicate HttpResponse.